docs/pages/update-spec/introduction.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Update Spec
+sidebar_title: EAS Update 0
+---
+
+version 0
+
+---
+
+## Introduction
+
+This is the specification for EAS Update a protocol for managing over the air updates to apps running on multiple platforms.
+
+### Conformance
+
+Conforming servers and clients must fulfill all normative requirements. Conformance requirements are described in this document via both descriptive assertions and key words with clearly defined meanings.
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in the normative portions of this document are to be interpreted as described in [IETF RFC 2119](https://tools.ietf.org/html/rfc2119). These key words may appear in lowercase and still retain their meaning unless explicitly declared as non‐normative.
+
+A conforming implementation of may provide additional functionality, but must not where explicitly disallowed or would otherwise result in non‐conformance.
+
+### Overview
+
+EAS Update is a protocol for managing over the air updates to an app running on multiple platforms. 
+In particular, it is a protocol for fetching an `update` where an `update` is defined as a [manifest](#manifest) together with the assets it references.
+
+An app running a conformant EAS Update client will load the most recent update saved by the client.
+
+In order to obtain an `update`:
+1. The client makes a [request](#request). 
+2. If the response body is a new manifest (checking the manifest ID is sufficient) it proceeds to download and store the assets referenced by the manifest.
+3. The client updates the local state according to any metadata provided by the response [headers](#headers)
+
+We anticipate the primary user of this spec will be companies who need to manage their own update server to satisfy internal requirements.
+
+## Request
+
+A conformant client MUST make a GET request with the following standard headers:
+
+```
+accept: application/expo+json,application/json
+expo-platform: string
+expo-runtime-version: string
+expo-channel-name: string
```

Comment:
I've been using lowercase headers since it's what modern HTTP/2 and 3 clients use now.

expo-channel-name is an EAS concept and likely should be removed from this spec.

It's fine for a client to accept just application/json but preferred for it to use expo+json so that servers can easily tell when to send back an Expo manifest. It's also okay for clients to accept other content types in addition to json (this will be important for non-JSON manifests).

Reply:
> expo-channel-name

So we've gone back and forth about what to do if a request is made without `expo-channel-name` on the EAS servers.

I'm thinking that if we are going to drop `expo-channel-name` as a requirement (makes sense) then we should move back to having the EAS server default to `main` instead of returning a 400.

Reply:
Talked with james and keeping it 400.